platforms/common/urlClient.h

```diff
@@ -23,16 +23,18 @@ class UrlClient {
     UrlClient(Options options);
     ~UrlClient();
 
-    UrlRequestHandle addRequest(const std::string& url, UrlCallback onComplete);
+    using Callback = std::function<void(UrlResponse&&)>;
```

Comment:
Why to create another alias here? Do not think `Callback` serves a different purpose than the `UrlCallback` defined in `platform.h`.